addon/components/swappable-container.js

```diff
@@ -1,7 +1,16 @@
 import Component from '@ember/component';
 import layout from '../templates/components/swappable-container';
+import { get } from '@ember/object';
 
 export default Component.extend({
     layout,
-    classNames: ['swappable-container']
+    classNames: ['swappable-container'],
+    didInsertElement() {
+        this._super(...arguments);
+        get(this, 'group.swappable').addContainer(this.$()[0]);
+    },
+    willDestroyElement() {
+        this._super(...arguments);
+        get(this, 'group.swappable').removeContainer(this.$()[0]);
```

Comment:
another jquery-less solution instead of `this.$().get(0)` that I think should work is `this.element`, might want to verify that though.